components/notes/notes.js

```diff
@@ -0,0 +1,27 @@
+import Link from "next/link";
+import BackButton from "../navigation/backbutton/backbutton";
+import { useRef, useEffect } from "react";
+
+export default function NoteFormular({}) {
+  const inputReference = useRef(null);
+  useEffect(() => {
+    inputReference.current.focus();
+  }, []);
+
+  return (
+    <form action="/send-data-here" method="post">
+      <textarea type="text" id="first" name="textarea" ref={inputReference} />
+      <label for="textarea">
+        <figcaption>Take A Note</figcaption>
+      </label>
+      <BackButton />
+      <Link
+        href={{
+          pathname: "/",
+        }}
+      >
+        <button type="button">end</button>
```

Comment:
If you want this button to trigger a form submit, the type should be `"submit"`. (Not sure if that's the idea...)

Reply:
oh, no this button just signals for the user, that this is the end of the route through "dailycard"... leading to the indexpage by clicking. but yes, i should add a submit button also!